CMakeLists.txt

```diff
@@ -11,7 +15,11 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 set(LIBDDWAF_SHARED_LINKER_FLAGS "-static-libstdc++" CACHE STRING "Shared library extra linker flags")
 set(LIBDDWAF_EXE_LINKER_FLAGS "" CACHE STRING "Executable extra linker flags")
 set(LIBDDWAF_PACKAGE_SUFFIX "" CACHE STRING "Suffix for packaging purposes")
-set(LIBDDWAF_PACKAGE_PROCESSOR ${CMAKE_SYSTEM_PROCESSOR} CACHE STRING "Alternative processor for packaging purposes")
+if(CMAKE_OSX_ARCHITECTURES MATCHES "arm64")
```

Comment:
I don't particularly like that, but I'm not well versed enough in `cmake` to improve this significantly.

Reply:
I don't think there's a better alternative. `OSX_ARCHITECTURES` doesn't seem to influence other variables. It would be difficult too, since, as the name suggests, there can be more than one architecture.

Reply:
I don't anticipate us building universal fat binaries, so I guess there would be only one ever for the foreseeable future.